src/main/java/decodes/sql/EngineeringUnitIO.java

```diff
@@ -215,30 +215,30 @@ public void write(EngineeringUnit eu)
 		write(eu, true);
 	}
 
-	/** 
-	  Write a single EngineeringUnit object to the SQL database. 
-	  @param eu the EU
-	  @param deleteFirst true if old object needs to be deleted first
-	*/
-	public void write(EngineeringUnit eu, boolean deleteFirst)
-		throws DatabaseException, SQLException
```

Comment:
If a block of code isn't needed anymore just delete it, it will live on in the commit history if we need it again.

Reply:
Done